specification/sdk-tracing.md

```diff
@@ -260,6 +260,40 @@ high contention in a very high traffic service.
 * `maxExportBatchSize` - the maximum batch size of every export. It must be
   smaller or equal to `maxQueueSize`. The default value is `512`.
 
+##### Span watcher processor
+
+This implementation passes spans that have been started but not yet ended
+to the configured exporter.
+On each interval defined by `reportIntervalMillis`, all unfinished spans that
+have been started more than `reportIntervalMillis` ago, are exported as is.
+
+**Configurable parameters:**
+
+* `exporter` - the exporter where the spans are pushed.
+* `maxQueueSize` - the maximum queue size. After the size is reached spans are
```

Comment:
I think it should be specified which spans to drop (e.g. oldest, new ones added,...)

Reply:
Indeed, this should be specified for the batch processor as well.
What do you think would be best?
I can imagine use cases for oldest (long-running), newest (the ones rushing in recently, causing the overflow) as well as random ones (would cover both).

Reply:
Easiest is to drop new spans. As there is no `correct` this should be fine.